hydrolib/core/dflowfm/extold/parser.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+from typing import Dict, List
+
+from hydrolib.core.dflowfm.extold.common_io import ORDERED_FORCING_FIELDS
+
+
+class Parser:
+    """Parser class for parsing the forcing data of the old external forcings file to a dictionary to construct the `ExtOldModel` with."""
+
+    @staticmethod
+    def parse(filepath: Path) -> Dict:
+        """Parses the file at the specified path to the forcing data.
+
+        If a line starts with an asterisk (*) it is considered a comment and will not be parsed.
+
+        Args:
+            path (Path): The path to parse the data from.
+            model_fields (List[str]): List of the  ordered model fields.
+
+        Returns:
+            Dict[str, List[Dict[str, str]]]: A dictionary containing the forcing data under the key 'forcing'.
```

Comment:
Can you add the error this method may raise to the documentation? (In case the properties are not in the correct order)

Reply:
Yes I totally forgot.